README.md

```diff
@@ -272,3 +272,11 @@ You can enable caching to the redis backend by supplying a redis URL
 
 -   `REDIS_URL`: The redis database URL to use for caching, e.g.
     `redis://some-host:6379/0`
+
+## Testing
+
+Tests can be run with this command: `docker-compose exec django pytest -cpytest.ini`
```

Comment:
Why is providing the `-c` flag required? Pytest should automatically use `pytest.ini`

Reply:
Looks like it does, somehow I was under the impression that it doesn't. Well everyday you learn something new :d I'll remove this.